.travis/release_stable.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
 MESSAGE=$(git log --format=%B -n 1 $TRAVIS_COMMIT)
+npm run verify
 git clone ${REPO}.git
```

Comment:
you'll need to checkout the stable branch when you clone the repo so that you get the correct Jenkinsfile.
`git clone ${REPO}.git -b stable`